src/box/lua/schema.lua

```diff
@@ -956,6 +966,7 @@ box.schema.space.alter = function(space_id, options)
         box.error(box.error.NO_SUCH_SPACE, '#'..tostring(space_id))
     end
     check_param_table(options, alter_space_template)
+    check_schema_version(options)
```

Comment:
I think it's better to add `_force` to every options template. Like you've done it for `alter_index_template`.
Otherwise there's no point in passing options to `check_schema_version`, if calling code doesn't allow `options._force`.

Reply:
Done